src/fidesops/analytics.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+from platform import system
+from typing import Dict
+
+from fideslog.sdk.python.client import AnalyticsClient
+from fideslog.sdk.python.event import AnalyticsEvent
+from fideslog.sdk.python.exceptions import AnalyticsError
+from fideslog.sdk.python.utils import FIDESOPS, generate_client_id
+
+from fidesops.core.config import config, update_config_file
+
+from ._version import get_versions
+
+logger = logging.getLogger(__name__)
+
+
+def get_version() -> str:
+    """Version of Fidesops"""
+    return get_versions()["version"]
+
+
+def in_developer_mode() -> bool:
+    """True if running in dev mode"""
+    return config.dev_mode
+
+
+def in_docker_container() -> bool:
+    """`True` if the command was submitted within a Docker container. Default: `False`."""
+    return True
+
+
+def running_on_local_host() -> bool:
+    """For events submitted as a result of making API server requests, `True` if the API server is running on the user's local host. Default: `False`."""
+    return False
+
+
+def generate_and_store_client_id() -> str:
+    update_obj: Dict[str, Dict] = {}
+    client_id: str = generate_client_id(FIDESOPS)  # get fideslog bytestring
```

Comment:
This comment threw me. My interpretation was "get the bytestring representing fideslog", not "get the fidesops bytestring from the fideslog SDK". Since the latter is clearly what you meant, and since `FIDESOPS` is being imported from the fideslog SDK above, I think this comment should just be removed.
```suggestion
    client_id: str = generate_client_id(FIDESOPS)
```